docs/Models/burgers-equation-model.md

```diff
@@ -0,0 +1,187 @@
+# Viscous Burgers Equation
+
+## Definition
+The [`SELF_Burgers1D_t` module](../ford/module/self_burgers1d_t.html) defines the [Burgers1D_t class](ford/type/burgers1d_t.html) class. In SELF, models are posed in the form of a generic conservation law
```

Comment:
```suggestion
The [`SELF_Burgers1D_t` module](../ford/module/self_burgers1d_t.html) defines the [`Burgers1D_t` class](ford/type/burgers1d_t.html). In SELF, models are posed in the form of a generic conservation law
```

Reply:
This has been resolved by committing the suggestion